.coveragerc

```diff
@@ -1157,6 +1157,10 @@ omit =
     homeassistant/components/uscis/sensor.py
     homeassistant/components/vallox/*
     homeassistant/components/vasttrafik/sensor.py
+    homeassistant/components/vconnex/__init__.py
+    homeassistant/components/vconnex/const.py
+    homeassistant/components/vconnex/entity.py
+    homeassistant/components/vconnex/switch.py
```

Comment:
Since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.